### DISCUSSION TITLE
epoptes in MX

### DISCUSSION THREAD

**Discussion by scamblor** [epoptes/epoptes#190] [General]
I have tried to install epoptes-client on a computer running MX linux. The document /etc/default/epoptes-client was not installed. I have installed it by hand. I run #epoptes-client -c and there is a successful response, but from the server the mx host is not seen (if there are other Debian computers)
That could be happening? How to fix them?

**Comment by alkisg** [comment]
Hi, `epoptes-client -c` fetches the certificate but it doesn't start epoptes-client.

To start epoptes-client, either run `epoptes-client` manually, or restart the client PC.

**Reply by scamblor** [reply]
I have epoptes-client as a service. Also I ran epoptes-client in the terminal and the host was not seen from the server.

**Comment by alkisg** [comment]
When you ran epoptes-client, what was the terminal output?

**Reply by scamblor** [reply]
Tomorrow I will seen it. It is in a school and today I can't see it. Thank you for your help

**Reply by scamblor** [reply]
the output is:
Epoptes-client connecting to 10.230.82.47:789...

**Comment by alkisg** [comment]
This means it can't reach the server or that it doesn't trust its certificate.
Go to the server, type the following, and upload the output:

```shell
sudo -i
hostname -I
ls -l /etc/epoptes
ufw status
```

**Reply by scamblor** [reply]
I tried it. the out in the server is
root@max10:~# hostname -I
192.168.1.100 
root@max10:~# ls -l /etc/epoptes
total 8
-rw-r--r-- 1 root root 1939 mar  1 18:38 server.crt
-rw------- 1 root root 3272 mar  1 18:38 server.key
root@max10:~# ufw status
Estado: inactivo

**Reply by scamblor** [reply]
If I typed:
systemctl status epoptes
The output is:
active (running)

**Comment by alkisg** [comment]
Your epoptes server has IP=192.168.1.100.
Your client is trying to connect to IP=10.230.82.47.

That's why it can't connect, it's trying to connect to another server.

**Reply by scamblor** [reply]
Now I triening with other pair of host server/client. The ip are 192.168.1.100 (server) and 192.168.1.102 (client)

**Comment by scamblor** [comment]
In this pair the output  of  tauping  epoptes-client int the client is:
Epoptes-client connecting to 192.168.1.100:789....
epoptes-client: 161: ./screenshot: not found
1 failed thumbnails so far!
epoptes-client: 161: ./screenshot: not found
2 failed thumbnails so far!
epoptes-client: 115: x11vnc: not found
epoptes-client: 161: ./screenshot: not found
3 failed thumbnails so far!
epoptes-client: 161: ./screenshot: not found
4 failed thumbnails so far!
epoptes-client: 161: ./screenshot: not found
5 failed thumbnails so far!
cepoptes-client: 161: ./screenshot: not found
6 failed thumbnails so far!
epoptes-client: 161: ./screenshot: not found
7 failed thumbnails so far!

**Comment by alkisg** [comment]
Do you want to come to the chat room for live troubleshooting?

https://ltsp.org/guides/chat-room/

Otherwise, paste the version of epoptes on the server, and epoptes-client on the client.

**Reply by scamblor** [reply]
Yes, I do,  thank you. But my english level is very low.
version of epoptes-client -> 21.02-1
verion of epoptes  0.5.10-2

**Comment by alkisg** [comment]
Add the epoptes PPA to your server, so that you get a recent e.g. 23.01 version instead of the ancient 0.5 one.

While on the client, you didn't use `--install-recommends` as mentioned in the installation documentation, so to fix it now, do:

```shell
sudo apt remove epoptes-client
sudo apt install --install-recommends epoptes-client
```
